esx_service/cli/vmdkops_admin.py

```diff
@@ -420,8 +420,8 @@ def commands():
                     'help': 'Volume to set options for, specified as "volume@datastore".',
                     'required': True
                 },
-                '--tenant': {
-                    'help': 'Name of the tenant the volume belongs to.',
+                '--vm-group': {
```

Comment:
TestVolumeCreationFirstTimeis related to old(er) ESX and Photon OS version only. Please check with @shuklanirdesh82 and make sure your testbed is updated.